WidgetKit/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.0.1</string>
```

Comment:
Why is this and the other pList change needed?

Reply:
That's explained by the previous PR comments. To fix the following issue:
![Screen Shot 2022-01-11 at 1 41 21 PM](https://user-images.githubusercontent.com/11338480/149002227-6920f463-991a-4a6f-ae43-b3afa3cb94aa.png)

the CFBundleShortVersionString of an app extension ('1.0') must match that of its containing parent app ('0.0.1').

Reply:
ahh I see. Okay great!